dex/src/main/scala/com/wavesplatform/dex/Matcher.scala

```diff
@@ -449,16 +449,10 @@ object Matcher extends ScorexLogging {
       implicit ec: ExecutionContext): FutureResult[(Asset, Int)] =
     getDescription(assetsDB, assetDesc)(asset).map(x => asset -> x.decimals)(catsStdInstancesForFuture)
 
-  private val waves = liftValueAsync[AssetsDB.Item](
-    AssetsDB.Item(
-      name = AssetPair.WavesName,
-      decimals = 8
-    ))
-
   private def getDescription(assetsDB: AssetsDB, assetDesc: IssuedAsset => Future[Option[BriefAssetDescription]])(asset: Asset)(
       implicit ec: ExecutionContext): FutureResult[AssetsDB.Item] =
     asset match {
-      case Waves => waves
```

Comment:
This way is better, because we don't create instances each time the asset is Waves.

Reply:
Done